src/dmd/astbase.d

```diff
@@ -281,7 +281,7 @@ struct ASTBase
         fwd,   // POD not yet computed
     }
 
-    enum TRUST : int
+    enum TRUST : ubyte
```

Comment:
actually I did not propose that because of [Iain said once](https://forum.dlang.org/thread/tacbjtzbkpuamuukgsgf@forum.dlang.org) this was not supported by GDC, this is not true anymore ?

Reply:
It is finally okay since a short while, Iain has already introduced the first enum classes, e.g.: https://github.com/dlang/dmd/blob/f9d1e63337b4aae0233968ed853c1bb9faa51564/src/dmd/declaration.h#L438

Reply:
> actually I did not propose that because of [Iain said once](https://forum.dlang.org/thread/tacbjtzbkpuamuukgsgf@forum.dlang.org) this was not supported by GDC, this is not true anymore ?

It is supported, but you can't use the type as a parameter or return type if you want C++98 compatibility.

GCC has shifted their baseline to C++11 now anyway, so we don't care about that anymore.

Reply:
cool, this should be used whenever possible, doesn't matter if that changes the size or not.